dell-c6320/workflows/dell-perccli-create-megaraid-graph.json

```diff
@@ -44,10 +50,24 @@
             }
         },
         {
+            "label": "refresh-catalog-driveid",
+            "taskName": "Task.Catalog.Drive.Id",
+            "waitOn": {
+                "refresh-catalog-megaraid": "succeeded"
+            }
+        },
+        {
+            "label": "finish-bootstrap-trigger",
+            "taskName": "Task.Trigger.Send.Finish",
+            "waitOn": {
+                "final-reboot": "finished"
```

Comment:
If this task waitOn final-reboot, you'd better to move it to be after final-reboot